src/main/java/nextstep/qna/domain/Answers.java

```diff
@@ -0,0 +1,55 @@
+package nextstep.qna.domain;
+
+import nextstep.qna.CannotDeleteException;
+import nextstep.users.domain.NsUser;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Answers {
+    private final List<Answer> answers;
+
+    public Answers(List<Answer> answers) {
+        this.answers = new ArrayList(answers);
+    }
+
+    public boolean hasOtherWriter(NsUser writer) {
+        if (hasNoAnswers()) {
+            return false;
+        }
+        return answers.stream()
+                .anyMatch(answer -> !answer.isOwner(writer));
+    }
+
+    private boolean hasNoAnswers() {
+        return answers.isEmpty();
+    }
+
+    public List<DeleteHistory> deleteAll(NsUser writer) throws CannotDeleteException {
+        List<DeleteHistory> deleteHistories = new ArrayList<>();
+        for (Answer answer : answers) {
+            answer.delete(writer);
+            deleteHistories.add(new DeleteHistory(ContentType.ANSWER, answer.getId(), answer.getWriter(), LocalDateTime.now()));
```

Comment:
Question 처럼 답변 삭제 이력은 Answer가 반환해주면 어떨까요?
```suggestion
            DeleteHistory deleteHistory = answer.delete(writer);
            deleteHistories.add(deleteHistory);
```